scripts/update_brew.sh

```diff
@@ -42,11 +42,7 @@ echo "Testing the formula locally with brew..."
 brew install Formula/story.rb
 story --version | grep $tag
 story --help
-git checkout -b release_$tag
 git commit -a -m "Release $tag."
```

Comment:
Imho this should be a new commit with amending:

```suggestion
git commit -m "Release $tag."
```